UnityProject/Assets/Scripts/Objects/Doors/FalseWallTileBlend.cs

```diff
@@ -47,14 +49,14 @@ private void Start()
 			UpdateWallSprite();
 			ChangeNearFalseWallSprites(GetNearFalseWalls(falseWallPosition));
 
-			UpdateManager.Add(CallbackType.LATE_UPDATE, UpdateMethod);
+			tileChangeManager.MetaTileMap.Layers[LayerType.Walls].onTileMapChanges.AddListener(UpdateMethod);
```

Comment:
is there no onTileMapChanges on the tileChangeManager.MetaTileMap?

Reply:
Or I can't find that or there is no such function